rembg/sessions/sam.py

```diff
@@ -142,7 +143,16 @@ def predict(
         Returns:
             List[PILImage]: A list of masks generated by the decoder.
         """
-        prompt = kwargs.get("sam_prompt", "{}")
+        prompt = kwargs.get(
+            "sam_prompt",
+            [
+                {
+                    "type": "point",
+                    "label": 1,
+                    "data": [int(img.width / 2), int(img.height / 2)],
+                }
+            ],
+        )
```

Comment:
Added default prompt
This simply selects the center of the image.
The existing one simply causes a validation error, but better than that.